s2i jboss sample command issue

I ran into issues with this command:
s2i build --ref=7.1.x --context-dir=helloworld https://github.com/jboss-developer/jboss-eap-quickstarts openshift/wildfly-101-centos7 test-jee-app

Details:

docker version:
Client:
 Version:      17.03.0-ce
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   60ccb22
 Built:        Thu Feb 23 10:40:59 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.03.0-ce
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   3a232c8
 Built:        Tue Feb 28 07:52:04 2017
 OS/Arch:      linux/amd64
 Experimental: true

go version:
go version go1.8 darwin/amd64

s2i version:
s2i v1.1.4-dirty

Error Message:
Branch 7.1.x set up to track remote branch 7.1.x from origin.
Switched to a new branch '7.1.x'
Found pom.xml... attempting to build with 'mvn package -Popenshift -DskipTests -B -s /opt/app-root/src/.m2/settings.xml'
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T16:41:47+00:00)
Maven home: /usr/local/apache-maven-3.3.9
Java version: 1.8.0_121, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.121-0.b13.el7_3.x86_64/jre
Default locale: en_US, platform encoding: ANSI_X3.4-1968
OS name: "linux", version: "4.9.12-moby", arch: "amd64", family: "unix"
[INFO] Scanning for projects...
[INFO] Downloading: https://repo1.maven.org/maven2/org/jboss/spec/jboss-javaee-7.0/1.0.4.Final-redhat-7/jboss-javaee-7.0-1.0.4.Final-redhat-7.pom
[INFO] Downloading: https://maven.repository.redhat.com/earlyaccess/all/org/jboss/spec/jboss-javaee-7.0/1.0.4.Final-redhat-7/jboss-javaee-7.0-1.0.4.Final-redhat-7.pom
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Non-resolvable import POM: Could not find artifact org.jboss.spec:jboss-javaee-7.0:pom:1.0.4.Final-redhat-7 in central (https://repo1.maven.org/maven2) @ line 102, column 25
[ERROR] 'dependencies.dependency.version' for javax.enterprise:cdi-api:jar is missing. @ line 115, column 21
[ERROR] 'dependencies.dependency.version' for org.jboss.spec.javax.annotation:jboss-annotations-api_1.2_spec:jar is missing. @ line 123, column 21
[ERROR] 'dependencies.dependency.version' for org.jboss.spec.javax.servlet:jboss-servlet-api_3.1_spec:jar is missing. @ line 130, column 21
@
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.jboss.quickstarts.eap:jboss-helloworld:7.1.0-SNAPSHOT (/opt/app-root/src/pom.xml) has 4 errors
[ERROR]     Non-resolvable import POM: Could not find artifact org.jboss.spec:jboss-javaee-7.0:pom:1.0.4.Final-redhat-7 in central (https://repo1.maven.org/maven2) @ line 102, column 25 -> [Help 2]
[ERROR]     'dependencies.dependency.version' for javax.enterprise:cdi-api:jar is missing. @ line 115, column 21
[ERROR]     'dependencies.dependency.version' for org.jboss.spec.javax.annotation:jboss-annotations-api_1.2_spec:jar is missing. @ line 123, column 21
[ERROR]     'dependencies.dependency.version' for org.jboss.spec.javax.servlet:jboss-servlet-api_3.1_spec:jar is missing. @ line 130, column 21
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
Aborting due to error code 1 from mvn package
ERROR: An error occurred: non-zero (13) exit code from openshift/wildfly-101-centos7

Thanks for your help!

--- a/README.md
+++ b/README.md
@@ -238,8 +238,8 @@ $ docker run --rm -i -p :8080 -t test-ruby-app
 ```
 
 ```
-$ s2i build --ref=7.1.x --context-dir=helloworld https://github.com/jboss-developer/jboss-eap-quickstarts openshift/wildfly-101-centos7 test-jee-app
-$ docker run --rm -i -p :8080 -t test-jee-app
+$ s2i build --ref=10.x --context-dir=helloworld https://github.com/wildfly/quickstart openshift/wildfly-101-centos7 test-jee-app
+$ docker run --rm -i -p 8080:8080 -t test-jee-app
 ```
 
 Want to know more? Read the following resources:

